src/sas/qtgui/Perspectives/Fitting/ComplexConstraint.py

```diff
@@ -137,7 +137,7 @@ def setupTooltip(self):
         Tooltip for txtConstraint
         """
         p1 = self.tab_names[0] + ":" + self.cbParam1.currentText()
-        p2 = self.tab_names[1] if len(self.tab_names) != 1 else self.tab_names[0] +"."+self.cbParam2.currentText()
```

Comment:
I am not sure we need to even modify this tooltip - it seems to be generated just once, at the widget instantiation and doesn't update on model change. We would probably hardcode a simple text and just present it.
Alternatively we could try updating this tooltip on model change...
What do you think?

Reply:
This is needed to show the same behaviour as for the standard case (when the parameters are set manually without the copy/paste), but it is true that a standard harcoded text will be enough.

Reply:
Yes, a hardcoded text seems enough, I don't think that having a tooltip that changes upon the models selected would be usefull as it is just an example. In fact, I had to do this change to satisfy a failing unit test, as my interpreter has the bad habit of exiting every time a unit test fails :-(